mmdet/models/losses/gfocal_loss.py

```diff
@@ -0,0 +1,107 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import LOSSES
+from .utils import weight_reduce_loss
+
+
+def quality_focal_loss(pred,
+                       label,
+                       score,
+                       weight=None,
+                       beta=2.0,
+                       reduction='mean',
+                       avg_factor=None):
+    # all goes to 0
+    pred_sigmoid = pred.sigmoid()
+    pt = pred_sigmoid
+    zerolabel = pt.new_zeros(pred.shape)
+    loss = F.binary_cross_entropy_with_logits(
+        pred, zerolabel, reduction='none') * pt.pow(beta)  # pt = pt.abs()
+    # find positive positions and quality labels
+    label = label - 1
+    pos = (label >= 0).nonzero().squeeze(1)
+    a = pos
```

Comment:
Assign `a = pos` seems to be unnecessary.

Reply:
Thanks for suggestions, I will update this part soon~